ui/app/templates/components/dashboard/secrets-engines-card.hbs

```diff
@@ -50,6 +50,7 @@
                 @route={{backend.backendLink}}
                 @model={{backend.id}}
                 class="has-text-weight-semibold is-no-underline"
+                target="_blank"
```

Comment:
Just confirming that we indeed want these links to always open in a [new tab](https://www.w3schools.com/tags/att_a_target.asp)? My understanding is you only do that if you're linking off your website. Ex: we were linking to documentation.

Reply:
I'll reconfirm with design and get back to you! The reason why we added this is that the breadcrumbs don't match the behavior of the link. Going to remove the new tab links for now.